backend/lib/github/webhookHandler.js

```diff
@@ -0,0 +1,52 @@
+//
```

Comment:
If we really want to have this in seperate files lets create the following strcutre:
```
github/webhook
  - index.js
  - router.js
  - handler.js
  - parser.js

```